php-checks/src/main/java/org/sonar/php/checks/PHPDeprecatedFunctionUsageCheck.java

```diff
@@ -138,4 +143,12 @@ private void checkAssertArguments(FunctionCallTree tree) {
     }
   }
 
+  private void checkDefineArguments(FunctionCallTree tree) {
+    SeparatedList<ExpressionTree> arguments = tree.arguments();
+    if (arguments.size() >= 3 && arguments.get(2).is(Kind.BOOLEAN_LITERAL) &&
+      "true".equalsIgnoreCase(((LiteralTree)arguments.get(2)).value())) {
```

Comment:
Extract `arguments.get(2)` into a variable `caseInsensitiveArgument`, it will be easier to read.